spec/lib/stat_generator_spec.rb

```diff
@@ -94,4 +94,25 @@
     expect(stats["Clojure"]).to eql 1
     expect(stats["Ruby"]).to eql 1
   end
+
+  it "disallows bad terms" do
+    ENV["HASHTAG_BLOCKED_TERMS"] = "iambad,soareyou,evildoers"
+    tweets = [tweet("#code2014 iambad #Ruby")]
+    stats = StatGenerator.new(tweets).generate
+    expect(stats["Ruby"]).to eql 0
+  end
+
+  it "disallows bad users" do
+    ENV["HASHTAG_BLOCKED_TERMS"] = "@iambad,@soareyou,@evildoers"
+    tweets = [tweet("#code2014 @iambad #Ruby")]
+    stats = StatGenerator.new(tweets).generate
+    expect(stats["Ruby"]).to eql 0
+  end
+
+  it "disallows bad hashtags" do
+    ENV["HASHTAG_BLOCKED_TERMS"] = "#iambad,#soareyou,#evildoers"
```

Comment:
This also should just be `ENV["HASHTAG_BLOCKED_TERMS"] = "iambad,soareyou,evildoers"`